boards/nxp/imx91_evk/imx91_evk_mimx9131_defconfig

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# ARM Options
+CONFIG_AARCH64_IMAGE_HEADER=y
+CONFIG_ARMV8_A_NS=y
+
+# MMU Options
+CONFIG_MAX_XLAT_TABLES=64
+
+# Cache Options
+CONFIG_CACHE_MANAGEMENT=y
+CONFIG_DCACHE_LINE_SIZE_DETECT=y
+CONFIG_ICACHE_LINE_SIZE_DETECT=y
+
+# Zephyr Kernel Configuration
+CONFIG_XIP=n
+CONFIG_KERNEL_DIRECT_MAP=y
```

Comment:
why can't SoC select/defconfig these?

Reply:
I think it's better not select the XIP and DIRECT_MAP configs under SoC, these configs can be different in boards with the same SoC.

Reply:
Currently there are some device drivers MMIO mapping uses direct map, so I also suggest to move it to SoC defconfig.

Reply:
Zephyr device drivers should use the virtual address, so not depends on this config. This is for some freestanding applications that access the MMIO/RAM using physical address, such as the applications in our Real-time-edge.

Reply:
So device driver are based on hal driver which need to use 1:1 mapping, so they are using K_MEM_DIRECT_MAP MMIP mapping flag, so depend on CONFIG_KERNEL_DIRECT_MAP.

Reply:
Already move it to SoC Kconfig and select it on condition HAS_MCUX.